db/migrate/20210707140230_create_accounts.rb

```diff
@@ -1,13 +0,0 @@
-class CreateAccounts < ActiveRecord::Migration[6.1]
-  def change
-    create_table :accounts do |t|
-      t.string :username, null: false
-      t.string :encrypted_password, null: false
-      t.string :email, null: false
-      t.references :role, null: false, foreign_key: true
-
-      t.timestamps
-    end
-    add_index :accounts, :username, unique: true
-  end
-end
```

Comment:
Because this project has been doing by only you, you could delete the migration file without issue. 
In fact, we have to create a migration to drop a table to ensure the DB is consistently with the member team.

Reply:
Ok I understand